### ISSUE TITLE
convert_from_path not creating files. Issue with latest version 1.12.0

### ISSUE THREAD

**Issue by khadeerusman** [Belval/pdf2image#127]
**Issue with latest version. 1.12.0 while the previous version works fine 1.11.0**

convert_from_path not working. Its running in background but it didn't display generated files. 

```
from pdf2image import convert_from_path
images = convert_from_path(r"D:\file.pdf",output_folder=r"D:\folder\",fmt="ppm",dpi=300)

```
**The above code didn't convert.Even using temporary path.**

```
import tempfile
from pdf2image import convert_from_path
temp_path=r"D:\\Folder"
with tempfile.TemporaryDirectory() as temp_path:
    images_from_path = convert_from_path("file.pdf", output_folder=temp_path,dpi=300)

```
Please check and fix it.

**Comment by Belval** [comment]
Just to be clear, when downgrading to `pdf2image==1.11.0` on that very machine, right now, it works?

It's possible that there is a regression since the tests don't cover Windows. I'll test it when I get home.

**Comment by Belval** [comment]
I was able to reproduce the issue, I will write a patch ASAP.

**Comment by Belval** [comment]
Should be fixed in version 1.12.1 please confirm.
